src/components/navbar/navbar.component.tsx

```diff
@@ -94,6 +106,39 @@ const Navbar: React.FC<NavbarProps> = ({
                 >
                   <UserAvatarFilledAlt20 />
                 </HeaderGlobalAction>
+                <HeaderGlobalAction
```

Comment:
Just use a single one. Also all the labels etc. are not right.

Reply:
Sir, right now I'm implementing the click even on the buttons and not the icons. If I use one button I'll implement the even on the Icons. So Clicking any where on the button asides the icon will not work.